src/components/AlbumCards/Album/CoverImage.jsx

```diff
@@ -0,0 +1,12 @@
+import { IconButtons } from "./IconButtons/IconButtons"
+
+export const CoverImage = ({ image, albumUrl }) => {
+    // Functional component that destructures the prop image, and sets it as the source of the image tag.
+    // Iconbuttons are added in, the styling for making them appear on hover is found under album-card.css
+    return (
+      <div className="cover-container">
+        <a href={albumUrl.spotify}><img className="cover-image" src={image.url} alt="Album Cover"/></a>
```

Comment:
I appreciate the little detail of adding an URL link to the album cover. This was not even mentioned in the requirements. Although as a user it's very intuative to click the album to be redirected to the album-site.